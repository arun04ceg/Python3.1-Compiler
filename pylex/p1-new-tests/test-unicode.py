# Unicode Testing
s = "a\xac\u1234\u0061\U00008000"
