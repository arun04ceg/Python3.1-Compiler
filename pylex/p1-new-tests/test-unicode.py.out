(NEWLINE)
(ID "s")
(PUNCT "=")
(LIT "a\xac\u1234\u0061\U00008000")
(NEWLINE)
(ENDMARKER)
