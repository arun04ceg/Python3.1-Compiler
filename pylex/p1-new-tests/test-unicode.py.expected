(ID "s")
(PUNCT "=")
(LIT "a¬\\u1234\\u0061\\U00008000")
(NEWLINE)
(ENDMARKER)
